.gitignore/test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] [REQUIRED] As stated in the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), you should add a `.gitignore` file listing a `test.md file` and `node_modules` directory. In your case, you have added a .gitignore folder instead of a file. After creating the .gitignore file, just list the following;
```
node_modules/
test.md
```
and they should be ignored by git.